### DISCUSSION TITLE
Deploying custom built client

### DISCUSSION THREAD

**Discussion by PlanosInformatica** [rustdesk/rustdesk#7213] [Q&A]
So i followed the documentation, managed to build an executable with `cargo run --release`, but the executable just works when it is ran with cargo run, so it is not feasable to deploy, double clicking the executable in the target/release just open a blank window the documentation seem a bit scarce(or pretty much non-existent) on how to make a deployable executable, so anyone could give some advice?

**Comment by tavelcom** [comment]
I have same problem

**Reply by PlanosInformatica** [reply]
From a bit of research, i found out that reading the github actions have all it is need to be done, sadly the community of this project seems a bit iffy answering things.

**Comment by angela-d** [comment]
If you use Github actions to custom build Rust Desk, you need developer signing keys setup in your repo or your apps won't run.
See this thread: https://github.com/rustdesk/rustdesk/discussions/3683
